pages/common/basename.md

```diff
@@ -1,11 +1,15 @@
 # basename
 
-> Returns non-directory portion of a pathname.
+> Removes leading directory portions from a path.
 
 - Show only the file name from a path:
 
 `basename {{path/to/file}}`
 
+- Show only the rightmost directory name from a path:
+
+`basename {{path/to/directory/}}`
```

Comment:
In my tests: the trailing `/` is not needed.
```suggestion
`basename {{path/to/directory}}`
```

Reply:
It is indeed not needed, but I found it well-suited for an example because the trailing `/` is removed in basename's output (which a novice user might not expect).

Reply:
Let's keep this as-is then.